examples/misc/lib/effective_dart/usage_good.dart

```diff
@@ -338,14 +338,15 @@ class Treasure {
 }
 
 class C {
-  double left, right, top, bottom, minTime;
+  num left, right, top, bottom, minTime;
   Point center;
   Map<Chest, Treasure> _opened;
 
   // #docregion use-arrow
-  double get area => (right - left) * (bottom - top);
+  num get area => (right - left) * (bottom - top);
```

Comment:
I get the impression that this is representing a game board, so I would have _expected_ this to be an `int`, I think...

Reply:
It's calculated from doubles (if I change the type that's on line 341).

Reply:
Right, I mean we're discussing an imaginary app here, so it's hard to do a code review ;) but I could see both these being `int` if they represent locations in a game. But we're into a nit on top of a nit now!